gandy/tests/test_models/test_gans.py

```diff
@@ -0,0 +1,200 @@
+"""Testing functions for UncertaintyModel gan class."""
+
+# import numpy as np
+import unittest
+import unittest.mock as mock
+
+# import deepchem
+
+import gandy.models.gans as gans
+import gandy.models.models
+
+
+class TestGAN(unittest.TestCase):
+    """Test GAN class."""
+
+    def test_inheritence():
+        """Ensure the subclass class inherits from parent class."""
+        assert issubclass(gans.gan, gandy.models.models.UncertaintyModel)
+
+    def test__build(self):
+        """
+        Test build function.
+
+        The build function should create a generator and discriminator.
+        This checks both functions are called. It also checks that both
+        generator and discriminator are attributes with type == Keras model.
+        """
+        # CHECK (normal) GAN
+        # create gan instance
+        subject = gans.GAN(xshape=(4,), yshape=(2,))
+        kwargs = dict(noise_shape=(5,))
+        subject._build(kwargs)
```

Comment:
`**kwargs` is required for python to parse out a dictionary. Like this python sees `subject._build({'noise_shape': (5,)})` instead of `subject._build(noise_shape=(5,))`